src/platforms/android/androidcontroller.cpp

```diff
@@ -149,19 +149,6 @@ void AndroidController::activate(const InterfaceConfig& config,
   foreach (auto item, config.m_allowedIPAddressRanges) {
     allowedIPs.append(IPAddress(item.toString()));
   }
-  foreach (auto addr, config.m_excludedAddresses) {
```

Comment:
Just digging in here and curious: why this change? I didn't think captive portal was enabled on Android. 
https://github.com/mozilla-mobile/mozilla-vpn-client/blob/89c56f10db8b11d0045b9da01f734929b2d19275/src/featurelistcallback.h#L59C1-L78

Reply:
We are getting rid of `m_excludedAddresses` entirely and keeping this breaks the Android build